tests/data/database_model.csv

```diff
@@ -0,0 +1,15 @@
+Attribute,Description,Valid Values,DependsOn,Properties,Required,Parent,DependsOn Component,Source,Validation Rules,Primary Key
```

Comment:
Is there a readme that describes what each of the `tests/data/*` files are used for? I can look at the data and see what's contained in it, but at a glance `database_model.csv` is not descriptive/specific enough.